hypothesis-python/RELEASE.rst

```diff
@@ -0,0 +1,6 @@
+RELEASE_TYPE: minor
+
+Enable :doc:`Ghostwritten <ghostwriter>` to accept a :func:`~python.__builtins__.classmethod` or :func:`~python.__builtins__.staticmethod` as input.
+:func:`~hypothesis.extra.ghostwriter.magic` can discover :func:`~python.__builtins__.classmethod` or :func:`~python.__builtins__.staticmethod` of classes through accepting a module or a class as input.
```

Comment:
Oddly enough, `classmethod` and `staticmethod` are in fact types, so I'd suggest just writing a bare:

```rst
:obj:`classmethod`
```

and seeing if that links properly.

Reply:
Thanks! It is a bit tricky